setup.py

```diff
@@ -2,7 +2,10 @@
 import setuptools
 
 try:
-    ret = subprocess.check_output("git describe --tags --abbrev=0", shell=True,)
+    ret = subprocess.check_output(
```

Comment:
What's with the change here?

Reply:
Ah. `black` was updated 8 hours ago. We don't lock the version of black. Since that function call ends with a `,`, it automatically becomes multi-line.

Reply:
I can move that change to a new pr.

Reply:
Thanks.